### PR TITLE
feat: change input of name TNS register

### PR DESCRIPTION
**Pull request by VishalRakholiya-iView**

### CODE REVIEW

File: packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx
Comment (clegirar) [inline packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx:1]: You don't create a specific component, you just copy what they done in `CreateDAOSection.tsx` but it's better to have an input component that check available names, that call the `TextInputCustom` etc. and called this component in `CreateDAOSection` and `TNSNameFinderModal`

**Review by clegirar — CHANGES_REQUESTED** [review]

File: packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx
Comment (VishalRakholiya-iView) [reply packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx:1]: you mean use `TextInputCustom` in both file (`CreateDAOSection` and `TNSNameFinderModal`)?

and write logic in `TextInputCustom` ?

**Review by n0izn0iz — CHANGES_REQUESTED** [review]
I agree with clemsss, please extract the component

File: packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx [hunk 1]
```diff
@@ -105,6 +110,19 @@ export const TNSNameFinderModal: React.FC<{
...
+  const networkId = network?.id;
+  const cosmosNetwork = getCosmosNetwork(networkId);
+  const normalizedTokenId = (
+    name + cosmosNetwork?.nameServiceTLD || ""
+  ).toLowerCase();
+
+  const tokenId = name + cosmosNetwork?.nameServiceTLD || "";
+
+  const { nsMintPrice: price } = useNSMintPrice(networkId, normalizedTokenId);
+  const { nameAvailable, loading } = useNSMintAvailability(networkId, tokenId);
```
Comment (n0izn0iz) [inline packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx:124]: all this should be inside the `AvailableNamesInput` component

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

File: packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx (hunk 1 above)
Comment (clegirar) [reply packages/components/modals/teritoriNameService/TNSNameFinderModal.tsx:124]: Yep you have to move the logic too in the component, if it's possible, but i think it is

**Review by clegirar — CHANGES_REQUESTED** [review]
When you moved the logic in the new component, please clean comments in the code

File: packages/screens/Organizations/components/CreateDAOSection.tsx [hunk 2]
```diff
@@ -7,19 +7,20 @@ import { RadioDescriptionSelector } from "./RadioDescriptionSelector";
...
 import { useNSAvailability } from "@/hooks/useNSAvailability";
 import { useSelectedNetworkInfo } from "@/hooks/useSelectedNetwork";
-import { getCosmosNetwork, NetworkKind } from "@/networks";
+import { NetworkKind } from "@/networks";
 import {
   neutral33,
   neutral77,
-  primaryColor,
-  redDefault,
+  // primaryColor,
```
Comment (clegirar) [inline packages/screens/Organizations/components/CreateDAOSection.tsx:19]: Just remove this comments

**Review by clegirar — CHANGES_REQUESTED** [review]

**Comment by clegirar** [comment]
And please pass the CI, don't export `TextInputCustomProps`

**Comment by clegirar** [comment]
The only thing i noticed in the UI is the equivalent in `$`
Screenshot 2024-02-22 132129
This one is from `create-org` screen
Screenshot 2024-02-22 132139
This one is from `tns`

Please add the equivalent in dollars in the custom component you created 👍

**Review by clegirar — APPROVED** [review]
LGTM 👍

**Pull request merged**